packages/synapse-react-client/src/lib/utils/SynapseClient.ts

```diff
@@ -2894,6 +2896,35 @@ export const getAllAccessRequirements = (
   return getAllOfPaginatedService(fn)
 }
 
+/**
+ * Given an array of access requirement IDs, return the IDs sorted by the user's status, where
+ * completed access requirements are shown first.
+ * @param accessToken
+ * @param requirementIds
+ */
+export const sortAccessRequirementsByCompletion = async (
```

Comment:
The analog to this was in AccessRequirementsList v1. Moved to SynapseClient and changed to return an array of IDs. Think this should probably be moved to an ARListUtils.ts file